src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-views/psbt-decoded-request-view-toggle.tsx

```diff
@@ -0,0 +1,25 @@
+import { FiSettings } from 'react-icons/fi';
+
+import { Box, Flex, Text, color } from '@stacks/ui';
+
+interface PsbtDecodedRequestViewToggleProps {
+  onSetShowAdvancedView(): void;
+  showAdvancedView: boolean;
+}
+export function PsbtDecodedRequestViewToggle({
+  onSetShowAdvancedView,
+  showAdvancedView,
+}: PsbtDecodedRequestViewToggleProps) {
+  return (
+    <Box px="loose">
```

Comment:
why not just add the padding to the flex?

Reply:
I think I took this from the drawer used in the other requests, let me look.